winch/codegen/src/masm.rs

```diff
@@ -1520,4 +1530,30 @@ pub(crate) trait MacroAssembler {
 
     /// If any bit in `src` is 1, set `dst` to 1, or 0 otherwise.
     fn v128_any_true(&mut self, src: Reg, dst: WritableReg) -> Result<()>;
+
+    /// Perform a vector add between `lsh` and `rhs`, placing the result in `dst`, where each lane
+    /// is interpreted to be `size` long.
+    fn v128_add(
```

Comment:
naming-wise, I'm completely sure what to call those. Maybe `vector_add` is more appropriate?

Reply:
I think the naming is fine here, as far as I can tell, we don't have any other `vector` naming convention.